client-v2/src/test/java/com/clickhouse/client/query/QueryTests.java

```diff
@@ -91,6 +89,10 @@ public class QueryTests extends BaseIntegrationTest {
 
     private boolean usePreallocatedBuffers = false;
 
+    static {
+//        System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "DEBUG");

```

Comment:
Lets remove it

Reply:
ok. this is only for current development. but will move into env settings on local